### DISCUSSION TITLE
Comparing results from BdTopo V2 and V3

### DISCUSSION THREAD

**Discussion by Denis-Vannier** [orbisgis/geoclimate#1001] [Q&A] [answered]
Hi !

Are there known issues when comparing results obtained when running the program with BDTopo V2.2 or with BDTopo V3 ? 

I would like to identify changes in climate zones over a ten-year period, on a metropolitan scale.

Thanks for your help !
Denis

**Comment by ebocher** [comment, marked as answer]
Hi @Denis-Vannier !

> Are there known issues when comparing results obtained when running the program with BDTopo V2.2 or with BDTopo V3 ?

No feedback to my knowledge


> I would like to identify changes in climate zones over a ten-year period, on a metropolitan scale.

This is a topic we are currently working on. Ping on my personnal email if you'd like to discuss it

Best
Erwan
